vhdbuilder/scripts/linux/mariner/tool_installs_mariner.sh

```diff
@@ -25,3 +25,7 @@ forceEnableIpForward() {
     net.bridge.bridge-nf-call-iptables = 1
 EOF
 }
+
```

Comment:
Can you also put a comment above the function what the workaround is about, so that the future readers of the code are aware